artifact-version-diff/src/main/java/io/quarkus/qe/PrepareOperation.java

```diff
@@ -58,13 +61,24 @@ public static void executeProcess(List<String> command, String errorMsg, Path pa
         try {
             Process process = builder.redirectErrorStream(true)
                     .directory(path.toFile())
-                    .redirectOutput(ProcessBuilder.Redirect.DISCARD)
-                    .redirectError(ProcessBuilder.Redirect.DISCARD)
                     .start();
+            StringBuilder output = new StringBuilder();
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    output.append(line).append("\n");
+                }
+            }
             process.waitFor();
+            if(process.exitValue() != 0) {
+                throw new RuntimeOperationsException(
+                        new RuntimeException("Process exited with code: " + process.exitValue()),
```

Comment:
Why this double wrapping here and below?

Reply:
That's valid only  when you catch an existing exception like in the case I mentioned it earlier today - see https://github.com/quarkus-qe/quarkus-utilities/pull/13/files#r1785783139 ... you were in ` } catch (IOException | InterruptedException e) { ...`